drivers/input/input_ite_it8801_kbd.c

```diff
@@ -0,0 +1,230 @@
+/*
+ * Copyright (c) 2024 ITE Corporation. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#define DT_DRV_COMPAT ite_it8801_kbd
+
+#include <zephyr/drivers/gpio.h>
+#include <zephyr/drivers/i2c.h>
+#include <zephyr/drivers/mfd/mfd_ite_it8801.h>
+#include <zephyr/drivers/pinctrl.h>
+#include <zephyr/input/input.h>
+#include <zephyr/input/input_kbd_matrix.h>
+
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(input_ite_it8801_kbd, CONFIG_INPUT_LOG_LEVEL);
+
+#define IT8801_MFDCTRL_COUNT IT8801_DT_INST_MFDCTRL_LEN(0)
```

Comment:
This assumes only one KBD driver installed.  This should be calculated on each KBD instance.

Reply:
Fixed, use multiple instances.